tests/gradients/parameter_shift/test_parameter_shift_shot_vec.py

```diff
@@ -2109,7 +2109,7 @@ def test_trainable_coeffs(self, broadcast, tol):
 
         res = dev.execute([tape])
         expected = -c * np.sin(x) * np.sin(y) + np.cos(x) * (a + b * np.sin(y))
-        assert np.allclose(res, expected, atol=tol, rtol=0)
```

Comment:
What happens when both are specified? Does one override the other?

Reply:
atol + rtol * abs(right value)

Reply:
I just updated a short intro in PR description

Reply:
Resolved since question answered.